src/lists/lists.ts

```diff
@@ -0,0 +1,46 @@
+import dotenv from "dotenv";
+import { AxiosResponse } from "axios";
+import { httpRequest } from "../httpMethod/httpRequest";
+import * as ResponseType from "./listDetails";
+
+dotenv.config();
+
+const sessionId = process.env.SESSION_ID;
+const listId = process.env.LIST_ID;
+
+export async function clearItemsFromList(): Promise<AxiosResponse> {
+  const url = `list/${listId}/clear`;
+  const queryParam: any = {
+    session_id: sessionId,
+    confirm: true,
+  };
+
+  try {
+    const response = await httpRequest<AxiosResponse>("POST", url, queryParam);
```

Comment:
httpRequest returns an object with AxioResponse type:
```
response: AxiosResponse {
  config: {},
  headers: {},
  data: {},
  status: 200
}
```
The `AxioResponse` interface you're passing as a type defines the type of `response.data`. In current implementation, you are defining:
```
response.data: AxiosResponse {
  config: {},
  headers: {},
  data: {},
  status: 200
}
```
`httpRequest<AxiosResponse<T>> `
![image](https://github.com/YahyoMahmudov/API-Testing-with-Jest/assets/62233209/89559806-7e00-4f41-978e-3c1bb01fbc81)